CHANGELOG.md

```diff
@@ -2,9 +2,16 @@
 
 # Versioning
 
-NB: This gem is versioned based on the `MAJOR`.`MINOR` version of rubocop. The first release of the
+ezCater is moving onto its own [Semantic Versioning](https://semver.org/) scheme starting with v1.0.0. All version bumps moving forward should increment using `MAJOR.MINOR.PATCH` based on changes.
```

Comment:
nitpick: perhaps `ezcater_rubycop` or `This gem` vs. `ezCater` as the subject here?